riva/clients/nmt/riva_nmt_streaming_s2t_client.cc

```diff
@@ -119,12 +124,12 @@ main(int argc, char** argv)
 
   std::signal(SIGINT, signal_handler);
   gflags::ParseCommandLineFlags(&argc, &argv, true);
-
+  emptyFileBeforeRequest();
   if (argc > 1) {
     std::cout << gflags::ProgramUsage();
     return 1;
   }
-
+  
   bool flag_set = gflags::GetCommandLineFlagInfoOrDie("riva_uri").is_default;
   const char* riva_uri = getenv("RIVA_URI");
 
```

Comment:
all changes in this file are not seen on the MR. Seems only partial changes pushed.